server/meshmodel/kubernetes/v1.25.2/v1.0.0/relationships/edge-network-statefulset-service.json

```diff
@@ -0,0 +1,87 @@
+{
+    "schemaVersion": "relationships.meshery.io/v1alpha2",
+    "version": "v1.0.0",
+    "kind": "Edge",
+    "metadata": {
+      "description": "A relationship that defines network edges between a StatefulSet and a Service",
+      "animation": "marching-ants",
+      "capabilities": {
+        "designer": {
+          "edit": {
+            "style": true,
+            "config": true,
+            "label": false
+          }
+        }
+      }
+    },
+    "model": {
+      "schemaVersion": "models.meshery.io/v1beta1",
+      "version": "v1.0.0",
+      "name": "kubernetes",
+      "model": {
+        "version": "v1.25.2"
```

Comment:
@MUzairS15, do we support `>`, the greater than operator, yet? If so, it would be ideal for @TomiwaAribisala-git to use the following, right?

```
"version": ">1.25.2"
```